src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");

```

Comment:
The message should indicate what type of `Machine` is working right now. In this case, it should be more informative and indicate that a `Bulldozer` is working.